src/nordsieck_utils.jl

```diff
@@ -0,0 +1,48 @@
+# This function computes the integral, from -1 to 0, of a polynomial
+# `P(x)` from the coefficients of `P` with an offset `k`.
+function ∫₋₁⁰dx(a, deg, k)
+  @inbounds begin
+    int = zero(eltype(a))
+    sign = one(eltype(a))
+    for i in 1:deg
+      int += sign * a[i]/(i+k)
+      sign = -sign
+    end
+    return int
+  end
+end
+
+# `l` is the coefficients of the polynomial `Λ` that satisfies conditions
+# Λ(0) = 1, Λ(-1) = 0, and Λ̇(-ξᵢ) = 0, where ξᵢ = (tₙ-tₙ₋₁)/dt.
+# It is described in the paper "A Polyalgorithm for the Numerical Solution
+# of Ordinary Differential Equations" by G. D. Byrne and A. C. Hindmarsh in
+# the page 86.
+# https://dl.acm.org/citation.cfm?id=355636
+function calc_coeff!(cache)
+  @inbounds begin
+    @unpack m, l, tau = cache
```

Comment:
unpack `tq` also as used in line `46`

Reply:
No, `tq` is changed in-place. Also, it is still WIP, so it is not ready for review.